tst/testbugfix/2019-07-17-cyclic-size.tst

```diff
@@ -0,0 +1,13 @@
+# Fix a bug that only occurs if all packages are loaded
+gap> F:=FreeGroup(2);
+<free group on the generators [ f1, f2 ]>
+gao> F:=F/[F.1/F.2];
```

Comment:
Maybe a stupid question, but why does it say gao here?

Reply:
That's not a stupid question, that's a stupid PR submitter *sigh*